### ISSUE TITLE
Expand PolicySetPolicy

### ISSUE THREAD

**Issue by richardszalay** [ewerkman/plumber-sc#12]
PolicySet's that are used Environment are a bit of a black box in the current UI. (eg. "ListShardingPolicy" doesn't show up, despite `Entity-PolicySet-ShardingPolicySet` being included)

I can see two ways of solving this:

1. Expand the policy set policies, but still keep the original PolicySetPolicy for reference
2. Include Policy Set's in the environment's dropdown and have referenced PolicySetPolicies link to that

**Comment by ewerkman** [comment]
Yes, the policy set are a little weird. Did you see if there is REST call to retrieve them?

**Comment by richardszalay** [comment]
So... there are two APIs: `PolicySets()` and `PolicySets('SomePolicy')`. The specific one works, but the "list all" API doesn't bring back any results. 

I'm still investigating but it looks like it might be related to whether the policies were added to the right managed list.

Ideally we can just pull back all of the policies, but it looks like it's possible to bring them back one-by-one as a fallback.

**Comment by ewerkman** [comment]
Did you get the specific one to work with anything other than the 'GlobalCartPolicies' sample?

**Comment by richardszalay** [comment]
Oddly, no. I wonder if it's accessing the right database?

**Comment by ewerkman** [comment]
Ok, just figured out what we need to do and you were right, it's about the database (or better environment). If you leave out the environment and the shopname header you get the global environment and all of the other policysets are in the global database.

So now I can get to the policysets.

**Comment by richardszalay** [comment]
Fantastic! I might try to tackle the friendly names one
